Index: i3ipc-python-2.1.1/Makefile
===================================================================
--- i3ipc-python-2.1.1.orig/Makefile
+++ i3ipc-python-2.1.1/Makefile
@@ -1,20 +1,7 @@
-.PHONY: test format lint all clean publish docs coverage docker-test
+.PHONY: all clean docs coverage
 .DEFAULT_GOAL := all
 
-source_dirs = i3ipc test examples
-
-lint:
-	flake8 $(source_dirs)
-
-format:
-	yapf -rip $(source_dirs)
-
-test:
-	./run-tests.py
-
-docker-test:
-	docker build -t i3ipc-python-test .
-	docker run -it i3ipc-python-test
+source_dirs = i3ipc examples
 
 clean:
 	rm -rf dist i3ipc.egg-info build docs/_build
@@ -22,7 +9,6 @@ clean:
 
 publish:
 	python3 setup.py sdist bdist_wheel
-	python3 -m twine upload --repository-url https://upload.pypi.org/legacy/ dist/*
 
 docs:
 	sphinx-build docs docs/_build/html
@@ -30,4 +16,4 @@ docs:
 livedocs:
 	sphinx-autobuild docs docs/_build/html --watch i3ipc -i '*swp' -i '*~'
 
-all: format lint docker-test
+all:
